specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2016-06-01/registeredidentities.json

```diff
@@ -284,18 +294,22 @@
       "type": "object",
       "properties": {
         "name": {
-          "type": "string"
+          "type": "string",
+          "readOnly": true
         },
         "type": {
-          "type": "string"
+          "type": "string",
+          "readOnly": true
         },
         "id": {
-          "type": "string"
+          "type": "string",
+          "readOnly": true

```

Comment:
Before I approve, I'm going to need to check to ensure that this doesn't cause breaking changes in any of the known generated languages.